package.json

```diff
@@ -27,13 +35,17 @@
     "jest-fetch-mock": "^3.0.3",
     "prettier": "^2.0.5",
     "rimraf": "^3.0.2",
+    "rollup": "^2.39.1",
+    "rollup-plugin-license": "^2.2.0",
+    "rollup-plugin-sizes": "^1.0.3",
+    "rollup-plugin-terser": "^7.0.2",
     "sinon": "^9.0.2",
     "ts-jest": "^26.1.3",
     "typedoc": "^0.17.8",
     "typescript": "^3.9.7"
   },
   "dependencies": {
-    "@fastly/performance-observer-polyfill": "^1.1.1",
+    "@fastly/performance-observer-polyfill": "^2.0.0",
```

Comment:
👉 New stuff!

Reply:
Ah yes, apologies; I should have documented this in the PR description. v2 of the polyfill doesn't introduce any new functionality, however it changes the compilation and output from CommonJS to ESM which makes it easier to consume and bundle in upstream projects such as this - which is why it required a major "breaking change" release. See the notes here: https://github.com/fastly/performance-observer-polyfill/releases/tag/v2.0.0